### DISCUSSION TITLE
React Peer Dependencies

### DISCUSSION THREAD

**Discussion by fabiandecuveland** [JiLiZART/BBob#150] [Q&A] [answered]
Hello!
I was just about to install this to replace [bbcode-to-react](https://www.npmjs.com/package/bbcode-to-react) in a React 17 codebase, but unfortunately your packages peer dependencies only seem to allow either React 15 or 18. If you could establish support for React 18 so easily, shouldn't React 16 and 17 work as well in the BBob versions below 2.8.1?
Is it possible to update the peer deps of some older package versions?

**Comment by JiLiZART** [comment, marked as answer]
Hi! I think I can tune up package.json peer deps to support variety of react versions

**Reply by JiLiZART** [reply]
You can track this PR https://github.com/JiLiZART/BBob/pull/151

**Reply by fabiandecuveland** [reply]
Cool, thanks for the quick response and PR!

**Reply by fabiandecuveland** [reply]
Any estimate for the release that includes this?

**Reply by JiLiZART** [reply]
Already there https://github.com/JiLiZART/BBob/releases/tag/v2.8.2
